src/electron/menu.ts

```diff
@@ -299,7 +299,12 @@ export function createMenu(): void {
 						}
 					})(),
 					click: () => {
+						if (store.getSelectedSlot()) {
```

Comment:
since this is the delete action it just means that the selected element is not to be deleted when in reality not an element but a slot of that element is selected in the element-list. and you can't delete slots.

Reply:
are you talking about text fields in the preview or in the main browser?
i tested this with the pattern search and property pane inputs. They still work as expected

Reply:
are you talking about text fields in the preview or in the main browser?
i tested this with the pattern search and property pane inputs. They still work as expected

Reply:
Looks like it was changed in one of the last electron updates. Earlier we had to specifically call the default behaviour when we did not want to do anything our self on delete.